docs/Concepts/Privacy/Multi-Tenancy.md

```diff
@@ -0,0 +1,24 @@
+description: Multi-tenancy
+<!--- END of page meta data -->
+
+# Multi-tenancy
+
+By default, each participant in a privacy network uses its own Besu and Orion
+node.
+
+Multi-tenancy allows multiple participants to use the same Besu and
+Orion node. Each participant is called a Tenant, and the Tenant Owner is the
+owner of the Besu and Orion node.
+
+!!! important
+    The tenant owner is responsible for [configuring multi-tenancy](../../Tutorials/Privacy/Configuring-Multi-Tenancy.md), and has 
+    access to all tenant data.
+    
+![Multi-tenancy](../../images/Multi-tenancy.png)
```

Comment:
Somewhere we need to mention that when using multi-tenancy that the Orion client API is only accessible to the multi-tenant Besu node otherwise other tenants could access the data.

Reply:
Done.